pisa/stages/osc/pi_2nu.py

```diff
@@ -0,0 +1,143 @@
+"""
+PISA pi stage for the calculation osc. probabilities assuming two-neutrino model
+
+"""
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+from numba import guvectorize
+
+from pisa import FTYPE, TARGET
```

Comment:
```python
from pisa import FTYPE, ITYPE, TARGET
```
Import `ITYPE` here to cast integers to the correct type in your function calls below